specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,165 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../document-status.md)
+
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Page View](#page-view)
+- [Ajax](#ajax)
+- [Exception](#exception)
+- [UserAction](#useraction)
+- [PageNavigationTiming](#pagenavigationtiming)
+- [AjaxTiming](#ajaxtiming)
+- [ResourceTiming](#resourcetiming)
+
+<!-- tocstop -->
+
+</details>
+
+
+This document describes the semantic conventions for browser events. The events may be represented either as Span Events or Log Events.
+
+All events have the following three high-level attributes. The domain has a fixed value of `browser`. The event name is specified at the beginning of each section below. The payload of the event goes in an attribute called `event.data` and the fields described below for each event go into this attribute.
+
+| Key  | Value | Description  |
+|---|---|---|
+| `event.domain` | browser | Fixed value |
+| `event.name` | | Described in each section below|
+| `event.data` | | A map of key/value pairs, with the keys for each event described in following sections|
+
+
+
+## Page View
+
+<!-- semconv browser.page_view -->
+The event name MUST be `page_view`.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
```

Comment:
@ramthi @MSNev 
- Added title
- Can you tell me the type, a brief description and an example for consentdetails?
- enduser.id is added to ResourceVarying since its common to all events, and can change in the same instance of a  Single Page Application

Reply:
Also, does userconsent require customer to hook the user consent response to the instrumentation library somehow? How is this information captured by the instrumentation library?

Reply:
For Microsoft this is a JSON encoded string, where the format is controlled by another team that manages consent. I don't have any example

Reply:
The user consent is a field that the application populates based on an additional SDK (for Microsoft anyway)

Reply:
Removing userconsent and consentdetails as these will have to added as custom attributes, since they cannot be added by auto-instrumentation.  @ramthi - this is based on discussion with @MSNev .